src/ci/resources/tes_application.conf

```diff
@@ -1,16 +1,65 @@
 include required(classpath("application.conf"))
```

Comment:
It looks like this is the config that's used for Centaur tests, so having it checked in in this form breaks the tests. Will adjust but keep this content easily available.